frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,10 +1,13 @@
 import React, { useEffect, useState } from "react";
 import { getSightings, Sighting } from "../../clients/apiClient";
 import { SightingCard } from "./SightingCard";
+// import {useNavigate} from "react-router-dom";
 import "./BrowseSightings.scss";
+import { MapPage } from "./MapPage";
```

Comment:
It's not a page - we're using the word "Page" to specifically mean those top-level components with `<h1>` headings and stuff, and never for components that get included inside other ones
```suggestion
import { SightingMap } from "./SightingMap";
```